tools/wptrunner/wptrunner/manifestupdate.py

```diff
@@ -320,10 +321,10 @@ def build_unconditional_tree(_, run_info_properties, results):
 
 
 class PropertyUpdate(object):
-    property_name = None
-    cls_default_value = None
-    value_type = None
-    property_builder = None
+    property_name = None  # type: ClassVar[str]
+    cls_default_value = None  # type: ClassVar[Any]
+    value_type = None  # type: ClassVar[type]
+    property_builder = None  # type: ClassVar[Callable[[Any, Any, Any], Any]]
```

Comment:
This type is no good. This will be set to either `build_conditional_tree` or `build_unconditional_tree` but I don't know how to capture that. It seems like a specific function isn't a type.